src/portable/raspberrypi/rp2040/dcd_rp2040.c

```diff
@@ -249,11 +246,28 @@ static void __tusb_irq_path_func(dcd_rp2040_irq)(void)
 {
     uint32_t const status = usb_hw->ints;
     uint32_t handled = 0;
+#if TUD_OPT_RP2040_USB_DEVICE_UFRAME_FIX
+    struct hw_endpoint *ep;
+#endif
 
     if (status & USB_INTF_DEV_SOF_BITS)
     {
       handled |= USB_INTF_DEV_SOF_BITS;
 
+#if TUD_OPT_RP2040_USB_DEVICE_UFRAME_FIX
+    hw_endpoint_lock_update(ep, 1);
```

Comment:
looks like indent here is wrong

Reply:
The whole file uses 2 and 4-space indents interchangeably throughout, but yes it looks off